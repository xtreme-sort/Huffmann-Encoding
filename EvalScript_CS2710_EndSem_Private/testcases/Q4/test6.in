GenerateLengthLimitedHuffman
nusrquzrbgitwuldoiqptushucdsnxalbaaqkenaiyoodgvozbrpulcpblediveezwlwwzkzkgiznrpcinlbtvldsvvmnuqpazqsvaqdexzltkkbjvpjpmthpaodauzjxybfdamiqcxcsamcohwoawtomesbupmhcneyrwvznepzosagwrwsjmtdmuiaprekfmqgogcaxisasbynuaupcpfutsfwqoksjoynwjlzpvxbouzdrludulpzsjbwxfyfyrzetaxvdzpjrogtkahqtmuxfqqrtcqrjfxskfxrxlverjmwciimxfbuncgleaauoihfrujfkkhqbsqwgdahczpafpbfmbohffaxdegglqgrlvpsgtzcfsnsgcirbadixzvqfuaedugvigmphsmiyzlcvbpjmkdqjfyyznjvlgndcjrvcrapxefvdbycmoahwhtecnmjhvcpmvzexuijpxmzxztpzkpejnuoapsdhtymcfombucnrhnprtrjdncrtdupmlnzfjmthqowewmdvgpnhpyledyjdqozlideqpdvbivwuuyqwatsvzngiomlqurewhgkhyimggvrtdvumwnyjjunpqfzgvyujnncdkmejvyvtqbhafmchddbzfticoeokfhjyqvxzlpolcgwutejzlwssqpwplwkwxieqqaoevcdijivgcqvamkwsqqssqjisetwfkrssmyhugvjvhnynjhhehttafngrzguulxooxuyutkvbnvwcfvobvmgbyemeitpkdpnkowntckv
8
Quit
