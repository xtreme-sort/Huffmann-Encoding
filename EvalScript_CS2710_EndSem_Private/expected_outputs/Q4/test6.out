d 0000
m 0001
p 0010
u 0011
v 0100
z 0101
a 01100
b 01101
c 01110
e 01111
f 10000
g 10001
h 10010
i 10011
j 10100
k 10101
l 10110
n 10111
o 11000
q 11001
r 11010
s 11011
t 11100
w 11101
x 11110
y 11111
cost: 3724
